broker/consumer/vm_startup.sh

```diff
@@ -41,41 +52,26 @@ KAFKA_TOPIC_DEFAULT="${kafka_topic_syntax/yyyymmdd/${yyyymmdd}}"
 KAFKA_TOPIC="${KAFKA_TOPIC_FORCE:-${KAFKA_TOPIC_DEFAULT}}"
 PS_TOPIC="${PS_TOPIC_FORCE:-${PS_TOPIC_DEFAULT}}"
 # set VM metadata, just for clarity and easy viewing
-gcloud compute instances add-metadata "$consumerVM" --zone "$zone" \
-    --metadata="PS_TOPIC=${PS_TOPIC},KAFKA_TOPIC=${KAFKA_TOPIC}"
-
-
-#--- Files this script will write
-fout_run="${workingdir}/run-connector.out"
-fout_topics="${workingdir}/list.topics"
-
-#--- Download config files from GCS (just grab the whole bucket)
-cd ${brokerdir}
-# remove all consumer files except the keytab
-find ${workingdir} -type f -not -name 'pitt-reader.user.keytab' -delete
-# download fresh files
-gsutil -m cp -r "gs://${broker_bucket}/consumer" .
-gsutil -m cp -r "gs://${broker_bucket}/schema_maps" .
-
-cd ${workingdir}
+gcloud compute instances add-metadata "${consumerVM}" --zone "${zone}" \
+    --metadata "PS_TOPIC=${PS_TOPIC},KAFKA_TOPIC=${KAFKA_TOPIC}"
 
 #--- Set the connector's configs (project and topics)
-fconfig=ps-connector.properties
-sed -i "s/PROJECT_ID/${PROJECT_ID}/g" ${fconfig}
-sed -i "s/PS_TOPIC/${PS_TOPIC}/g" ${fconfig}
-sed -i "s/KAFKA_TOPIC/${KAFKA_TOPIC}/g" ${fconfig}
+fconfig="ps-connector.properties"
+sed -i "s/PROJECT_ID/${PROJECT_ID}/g" "${fconfig}"
+sed -i "s/PS_TOPIC/${PS_TOPIC}/g" "${fconfig}"
+sed -i "s/KAFKA_TOPIC/${KAFKA_TOPIC}/g" "${fconfig}"
 
 #--- Check until alerts start streaming into the topic
-alerts_flowing=false
-while [ "${alerts_flowing}" = false ]
+alerts_flowing="false"
+while [ "${alerts_flowing}" = "false" ]
 do
     # get list of live topics and dump to file
     {
         /bin/kafka-topics \
-            --bootstrap-server public2.alerts.ztf.uw.edu:9094 \
+            --bootstrap-server "public2.alerts.ztf.uw.edu:9094" \
             --list \
-            --command-config ${workingdir}/admin.properties \
-            > $fout_topics
+            --command-config "${workingdir}/admin.properties" \
+            2>&1 > "${fout_topics}"
```

Comment:
seeing this for the first time (2>&1 command)! just looked up what it does on Stack Overflow

Reply:
It is a real thing, but actually I'm using it wrong here. #178 fixes it back to `&>> "${fout_topics}"`.